specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -478,6 +478,122 @@
         }
       }
     },
+    "/providers/Microsoft.ADHybridHealthService/addsservices/{serviceName}/features/{featureName}/userpreference": {
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets the user preferences for a given feature.",
+        "x-ms-examples": {
+          "addsServices_getUserPreference": {
+            "$ref": "./examples/UserPreference.json"
+          }
+        },
+        "operationId": "addsServices_getUserPreference",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "featureName",
+            "in": "path",
+            "description": "The name of the feature.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",

```

Comment:
Please provide a meaningful description.

Reply:
Added description text for all operations.